payment-stellar/src/main/java/org/stellar/anchor/paymentservice/stellar/util/NettyHttpClient.java

```diff
@@ -0,0 +1,31 @@
+package org.stellar.anchor.paymentservice.stellar.util;
+
+import io.netty.channel.ChannelOption;
+import io.netty.handler.codec.http.HttpHeaderNames;
+import io.netty.handler.codec.http.HttpHeaderValues;
+import io.netty.handler.timeout.ReadTimeoutHandler;
+import io.netty.handler.timeout.WriteTimeoutHandler;
+import java.time.Duration;
+import reactor.netty.http.client.HttpClient;
+
+public class NettyHttpClient {
```

Comment:
This is ok to check in. But we should create a subproject `payment-share` to place commonly used classes.

Reply:
Agreed. I also increased the request timeout from 30 to 60 though because Horizon returns 504 after 30 seconds. We should have a higher limit than 30 seconds to ensure we receive 504's and we don't perceive it as a connection failure.

Reply:
When we refactor, we will need to externalize the timeout configuration.